src/lib/keychain.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import debugLib from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import Insecure from './keychain/insecure';
+import type { Keychain } from './keychain/keychain';
+
+let exportValue: Keychain;
+const debug = debugLib( '@automattic/vip:keychain' );
+
+try {
+	// Try using Secure keychain ("keytar") first
+	const Secure = require( './keychain/secure' ) as Keychain & ( new() => Keychain );
```

Comment:
This line could use more love :) 

Shall we break it down to multiple lines. `& ( new() => Keychain );` looks pretty alien to me.

Reply:
Done.